_includes/layout/base/wallets-selector.html

```diff
@@ -0,0 +1,601 @@
+            </label>
+            <div class="sidebar-selector-info tooltip-trigger">
+              <img src="/img/icons/info-alt.svg" alt="info">
+              <div class="wallet-selector-tooltip tooltip">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+              </div>
+            </div>
+          </div>
+          <div class="important-checkbox-row">
+            <label class="checkbox-wrapper">
+              <input id="walletSelector" type="checkbox" name="features" value="mixing_shuffling" class="checkbox" />
+              <div class="checkbox-text">Mixing / Shuffling</div>
+            </label>
+            <div class="sidebar-selector-info tooltip-trigger">
+              <img src="/img/icons/info-alt.svg" alt="info">
+              <div class="wallet-selector-tooltip tooltip">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+              </div>
+            </div>
+          </div>
+          <div class="important-checkbox-row">
+            <label class="checkbox-wrapper">
+              <input id="walletSelector" type="checkbox" name="features" value="multisig" class="checkbox" />
+              <div class="checkbox-text">Multisig</div>
+            </label>
+            <div class="sidebar-selector-info tooltip-trigger">
+              <img src="/img/icons/info-alt.svg" alt="info">
+              <div class="wallet-selector-tooltip tooltip">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+              </div>
+            </div>
+          </div>
+          <div class="important-checkbox-row">
+            <label class="checkbox-wrapper">
+              <input id="walletSelector" type="checkbox" name="features" value="segwit" class="checkbox" />
+              <div class="checkbox-text">Segwit</div>
+            </label>
+            <div class="sidebar-selector-info tooltip-trigger">
+              <img src="/img/icons/info-alt.svg" alt="info">
+              <div class="wallet-selector-tooltip tooltip">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+              </div>
+            </div>
+          </div>
+        </div>
+        <h3 class="sidebar-selector-title">Need some help?</h3>
+        <p class="sidebar-selector-text">You can find more information and help on the resources and community pages or on the Wiki FAQ</p>
+      </div>
+
+      <div class="wallet-selector-content">
+        <div class="wallet-table-wrapper">        
+          <div class="wallet-table">        
+            <div class="wallet-table-row">
+              <div class="wallet-table-head">
+                <p class="wallet-table-title">iOS Wallets:</p>
```

Comment:
Let's make sure that this:

1. Updates dynamically based on the Operating System that is chosen, and,
2. Doesn't appear if there is no Operating System chosen yet

Reply:
Yes

Reply:
Thanks